package.json

```diff
@@ -73,6 +73,27 @@
     }
   },
   "contributes": {
+    "commands": [
+      {
+        "command": "kaoto.open",
+        "title": "Open with Kaoto Graphical Editor for Camel"
+      }
+    ],
+    "menus": {
+      "commandPalette": [
+        {
+          "command": "kaoto.open",
+          "when": "false"
+        }
+      ],
+      "explorer/context": [
+        {
+          "command": "kaoto.open",
+          "when": "resourceExtname == .yaml || resourceExtname == .yml",
+          "group": "1_belowOpenWith"
```

Comment:
in theory, it would be better placed in `navigation` group, but `navigation@x` with whatever x is giving random results and most of the time it is at the top which is not nice.